activerecord-postgis-adapter.gemspec

```diff
@@ -19,8 +19,8 @@ Gem::Specification.new do |spec|
 
   spec.required_ruby_version = ">= 2.5.0"
 
-  spec.add_dependency "activerecord", "~> 6.0"
-  spec.add_dependency "rgeo-activerecord", "~> 6.0"
+  spec.add_dependency "activerecord", "~> 6.1"
+  spec.add_dependency "rgeo-activerecord", "~> 6.2", ">= 6.2.2"
```

Comment:
I think we should just pin this to `>= 6.2.2`. Not sure that it's necessary to specify both here.